CHANGELOG.md

```diff
@@ -58,6 +58,8 @@
 
 - Fix regression when files `source`d from `%environment` contain `\` escaped
   shell builtins (fixes issue with `source` of conda profile.d script).
+- The `oci` commands will operate on systems that use the v2 unified cgroups
```

Comment:
Noticed after merge: this was put into the wrong place, under 3.8.2 instead of changes since last release.  Restored to proper place with issue apptainer/apptainer#11.